.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,42 @@
+| Q | A
+| --- | ---
+| Bug fix? | [yes|no]
+| New feature? | [yes|no]
+| BC breaks? | [yes|no]
+| Deprecations? | [yes|no]
+| Tests pass? | [yes|no]
```

Comment:
The problem is that it doesn't "trigger" anything, but only in a hypothetical process. People won't look at that field, and whatever is written here can't be trust.